cmd/arc_summary/arc_summary3

```diff
@@ -221,6 +222,28 @@ elif sys.platform.startswith('linux'):
 
         return descs
 
+def handle_unraisableException(exc_type, exc_value=None, exc_traceback=None,
+                               err_msg=None, object=None):
+   handle_Exception(exc_type, object, exc_traceback)
+
+def handle_Exception(ex_cls, ex, tb):
+    if ex is IOError:
+        if ex.errno == errno.EPIPE:
+            sys.exit()
+
+    if ex is KeyboardInterrupt:
+        sys._exit()
```

Comment:
just noticed this says `sys._exit()` lol

also, the check should be `if ex_cls is KeyboardInterrupt:` to catch this (apparently, tested on 3.7.3 and 3.9.2)

Reply:
"oops".

I believe it. I even suspect the IOError case would probably need to be ex_cls as well.